src/security/PeerOptions.cc

```diff
@@ -634,13 +634,14 @@ Security::PeerOptions::updateContextOptions(Security::ContextPointer &ctx)
     SSL_CTX_set_options(ctx.get(), parsedOptions);
 #elif USE_GNUTLS
     // NP: GnuTLS uses 'priorities' which are set only per-session instead.
+    (void)ctx;
 #endif
```

Comment:
Missing `#else` to mark `ctx` as unused when neither TLS library is enabled.

Reply:
right, thanks